net/ollama/Portfile

```diff
@@ -16,19 +15,17 @@ description         Ollama
 long_description    Ollama runs and manages LLMs
 homepage            https://ollama.com
 
-distfiles           ollama-darwin
-
-checksums           rmd160  703b9fd1cdaab4878fd1547bd4fe7a6c6f8d5040 \
-                    sha256  08a6dff6d22b756273874298f8ce1bfcf5812805e28743b43310c701dc3d56a4 \
-                    size    55334656
+depends_build       port:cmake \
+                    port:gcc14 \
```

Comment:
Why does this need gcc to build?

Reply:
good call. contrary to the developer docs, it doesn't.